docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.cs

```diff
@@ -27,7 +27,7 @@ public static void Example()
             // A pipeline for concatenating the age, parity and induced columns together in the Features column and training a KMeans model on them.
             string outputColumnName = "Features";
             var pipeline = ml.Transforms.Concatenate(outputColumnName, new[] { "Age", "Parity", "Induced" })
-                .Append(ml.Clustering.Trainers.KMeans(outputColumnName, numberOfClusters: 2));
+                .Append(ml.Clustering.Trainers.KMeansPlusPlus(outputColumnName, numberOfClusters: 2));
```

Comment:
```suggestion
                .Append(ml.Clustering.Trainers.KMeans(outputColumnName, numberOfClusters: 2));
```

Reply:
The algorithm that is  actually implemented  is [KmeansPlusPlus ](https://en.wikipedia.org/wiki/K-means%2B%2B) .   The underlying class is also called KMeansPlusPlus

This is also discussed here : https://github.com/dotnet/machinelearning/issues/2762#issuecomment-470229508

Reply:
The algorithm that is  actually implemented  is [KmeansPlusPlus ](https://en.wikipedia.org/wiki/K-means%2B%2B) .   The underlying class is also called KMeansPlusPlus

This is also discussed here : [https://github.com/dotnet/machinelearning/issues/2762#issuecomment-470229508](https://github.com/dotnet/machinelearning/issues/2762#issuecomment-470229508)

---
In reply to: [264484798](https://github.com/dotnet/machinelearning/pull/2903#discussion_r264484798) [](ancestors = 264484798)